rune/libenclave/internal/runtime/pal/spec.md

```diff
@@ -0,0 +1,27 @@
+# Enclave Runtime PAL API Specification
+
+## Introduction
+Enclave Runtime PAL API defines a common interface to interact between rune and enclave runtime.
+
+## Versions
+Enclave Runtime PAL API Specification currently support [PAL API v1](https://github.com/alibaba/inclavare-containers/blob/master/rune/libenclave/internal/runtime/pal/spec_v1.md) and [PAL API v2](https://github.com/alibaba/inclavare-containers/blob/master/rune/libenclave/internal/runtime/pal/spec_v2.md).
+
+If you want to use `rune` to run you enclave runtime, you can choose one of supported PAL API version to achieve.
+
+In the future, We well develop more PAL API version, If you have more needs on PAL API, please tell us and we will take it seriously.
+
+## Relationship of `rune` subcommands and PAL APIs
+`rune` have a lot of subcommand such as `create`,`start`,`exec`,`run` and etc. We will list the relationship of the subcommand and the PAL APIs they call.
```

Comment:
s/subcommand/subcommands/

Reply:
s/have/has/

Reply:
Change to:
s/We will list the relationship of the subcommand and the PAL APIs they call./List the relationship between them./